app/services/slack_username_updater.rb

```diff
@@ -0,0 +1,31 @@
+class SlackUsernameUpdater < SlackApiWrapper
+  def initialize(employee)
+    @employee = employee
+    configure_slack
```

Comment:
same comment as above -- any way we can put the slack ruby client config in one place?